src/vicsetup/tests/Makefile

```diff
@@ -0,0 +1,12 @@
+include ../defs.mak
```

Comment:
Shouldn't tests live under `tests/`? This way they get picked up by the CI pipeline.

Reply:
I struggled with this. These tests are for the **vicsetup** package and are only run manually at present. The tests directory runs tests against an installed **sgx-lkl**. And since the **vicsetup** tests use the **vicsetup** command-line tool, this assumes that the **vicesetup** tool has been installed as part of **sgx-lkl**. And installing the **vicsetup** tool invites usage and implies support.

Another alternative is for us to have a unit tests directory that can be run as part of the build. I'm not sure.

Reply:
See my comment https://github.com/lsds/sgx-lkl/pull/577#issuecomment-655594164.